tests/test_generator_declarative.py

```diff
@@ -1706,3 +1709,46 @@ class TestDomainJson(Base):
     foo: Mapped[Optional[dict]] = mapped_column(DOMAIN('domain_json', {domain_type.__name__}(astext_type=Text(length=128)), not_null=False))
 """,
     )
+
+
+# SQLAlchemy doesn't reflect
+@pytest.mark.skipif(
+    sys.version_info < (3, 10),
+    reason="This test assumes GeoAlchemy2 0.18.x and above, which does not support python 3.9",
+)
+@pytest.mark.parametrize("engine", ["postgresql"], indirect=["engine"])
+def test_geoalchemy2_types(generator: CodeGenerator) -> None:
+    Table(
+        "spatial_table",
+        generator.metadata,
+        Column("id", INTEGER, primary_key=True),
+        Column("geom", Geometry("POINT", srid=4326, dimension=2), nullable=False),
+        Column("geog", Geography("POLYGON", dimension=2)),
+    )
+
+    validate_code(
+        generator.generate(),
+        """\
+from typing import Any, Optional
+import typing
```

Comment:
This makes zero sense. Why are we using both `Any` and `typing.Any`? We should only be using `Any`.

Reply:
OK, I will consolidate this behavior with the rest of the code

Reply:
Consolidated this behavior. only `typing.Any` is imported, which matches the behavior for other non primite types (like the library treats uuids, as an exmaple)